build.sbt

```diff
@@ -41,14 +41,14 @@ lazy val borerAkkaSerializer = (projectMatrix in file("borer-akka-serializer"))
   .settings(commonSettings)
   .settings(
     libraryDependencies ++= Seq(
-        borerCore,
-        akkaTyped % Provided,
-        reflections,
-        borerDerivation % Test,
-        akkaTestKit % Test,
-        akkaStream % Test,
-        akkaStreamTestKit % Test,
-        enumeratum % Test))
+      borerCore,
```

Comment:
Whoops it keeps shuttling there and back... `sbt scalafmtSbtCheck`? See Hydra's Jenkinsfile, stage `Check formatting & code style`

Reply:
Plus project/.scalafmt.conf etc.

Reply:
Ok, I figured out what was happening. It seems my IDE is using different settings for formatting. It's a local fix.